modules/ROOT/pages/database-administration/standard-databases/create-databases.adoc

```diff
@@ -130,9 +130,14 @@ Defines credentials that need to be passed into certain seed providers.
 | `txLogEnrichment`
 | `FULL` \| `DIFF` \| `NONE`
 |
-Defines level of enrichment applied to transaction logs for Change Data Capture (CDC) purposes.
+Defines the level of enrichment applied to transaction logs for Change Data Capture (CDC) purposes.
 
 For details about enrichment mode, see link:{neo4j-docs-base-uri}/cdc/current/get-started/self-managed/#set-enrichment-mode/[Change Data Capture Manual -> Enable CDC on self-managed instances -> Set the enrichment mode].
+
+| `storeFormat`
+| `aligned` \| `standard` \| `high_limit` \| `block`
+|
+Defines the store format of a new database.
```

Comment:
We should also state that this is only effective if the database is created new. If the store is seeded with `seedURI`, or `existingDataSeedInstance` or if the command is being used to mount existing store files already on the disk they will use their current store format - there will not be any migration.